src/common/constants.ts

```diff
@@ -178,7 +176,7 @@ export const Constants = {
   CurrentVersion: "2018-06-18",
 
   SDKName: "azure-cosmos-js",
-  SDKVersion: version,
+  SDKVersion: process.env.VERSION,
```

Comment:
I'm guessing this is coming from Rollup? Can we have a PREFIX to make it clear this is coming from rollup and not something we expect users to set?

Reply:
Discussed with @christopheranderson offline. We'll need to use the approach on master to make this work in all environments. ESM (webpack) consumers won't get the SDK version set properly with rollup find/replace.